config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v0 do
-      resources :articles, only: [:index, :create]
+      resources :articles, only: [:index, :create, :show]
```

Comment:
you do not have a show action in the articles controller

Reply:
please remove it

Reply:
fixed